src/pty.zig

```diff
@@ -41,12 +41,14 @@ pub const Mode = packed struct {
 // a termio that doesn't use a pty. This isn't used in any user-facing
 // artifacts, this is just a stopgap to get compilation to work on iOS.
 const NullPty = struct {
+    pub const Errors = error{GetModeFailed};
```

Comment:
Idiomatically should be named `Error`

Secondly though I don't see this const (in any of the implementations) used anywhere. Is it?